# Multi-omics-Covid-19-vaccines
Multi-omics &amp; Covid-19 vaccines
Figure 2. A wider range of features are regulated after BNT162b2 than CoronaVac.  a, Scatter Plot Depicting Distribution of Differential Features. This plot illustrates the count of differential features within BNT162b2 or CoronaVac group, at baseline vs. 1-month post-vaccination (M1) and baseline vs. 6-month post-vaccination (M6). Each omics is positioned according to the direction of regulation, omics, and vaccine groups. The size of the circle represents the number of differential features in each respective comparison. The colors represent transcriptomics, metabolomics, and lipidomics, respectively.  b-e, Comparative Analysis of Vaccine-Induced Changes in genes, metabolites, lipids and cytokines. The Venn diagrams depict the number of shared and unique genes (b), metabolites (c), lipids (d), and cytokines (e) identified between pre-vaccination and post-vaccination timepoints for BNT162b2 and CoronaVac vaccines. The intersections indicate the count of features overlapping between different comparison sets. Metabolites, lipids, and cytokines were considered significant if the FDR was less than 0.1. For gene expression, significant differential features were defined by an FDR < 0.05 and an absolute log2 fold change (|log2(FC)|) > 1.5. A comprehensive dataset, including the complete number of features and the detailed statistical results, is available in the supplementary Source Data file (Supplementary Table 3-4).  

Figure 3. BNT162b2 is associated with heightened adaptive immunity and decreased neural functions and CoronaVac with heightened innate immunity. a. Bar plot for different Gene Ontology (GO) Biological Process. Each row in the chart corresponds to a different Gene Ontology (GO) Biological Process. The x-axis represents the count of occurrences or measurements associated with each GO term. For each GO term, there are two bars extending in opposite directions from a central axis at 0. The left panel represents a negative normalization enrichment score (suppression) and right panel indicates a positive normalization enrichment score (activation). Normalized enrichment scores calculated for the GO Pathway list for one-month post-vaccination vs. Baseline. The colors of the bars distinguish between two different vaccine groups: BNT162b2 (blue) and CoronaVac (red). b-c. Over-representation Analysis by GO list and Blood transcription module (BTM) analysis. These panels illustrate the comparative normalized enrichment scores (NES) derived from genes associated with Neural Function (b), and Immunological Functions (c) for pre-vaccination versus 1-month post-vaccination samples from BNT162b2 and CoronaVac recipients. For Cellular Processes, shared pathways between the two vaccines and the top 25 distinct pathways, as ranked by NES, are presented. The input data comprised variance-stabilized counts from DESeq2, which were processed using the GSVA function to calculate scaled NES. Dot size is proportional to the significance level (−log10 p-value), with the color indicating the NES. The significance of enrichment was determined using the fgsea package and adjusted for multiple testing. Complete source data are available as supplementary material. d. Distribution of Pathways with Positive Enrichment Post-Vaccination. The cumulative distribution function (CDF) plots illustrate the frequency of pathways exhibiting positive normalized enrichment scores (NES) when comparing pre-vaccination (M0) to 1-month post-vaccination (M1) samples for the BNT162b2 vaccine (left) and the CoronaVac vaccine (right). Pathways are categorized by their associated functions: Cellular Process (blue), Immunological Function (yellow), and Neural Function (red). The x-axis indicates the number of pathways showing positive effect sizes, while the y-axis displays the cumulative frequency of these pathways within the dataset. The dashed horizontal line at 0.5 marks the median of the distribution. e. Over-representation Analysis of Significantly Altered Metabolites by Vaccination (FDR <0.1). The bar plot displays enriched longitudinal differences in metabolite levels between baseline and 1-month post-vaccination. Bar plots are categorized by vaccine type, with BNT162b2 (left) and CoronaVac (right). The color intensity is proportional to the significance level.  

Figure 4 Longitudinal Profiling of Vaccine-induced Omics Features and Pathways Alterations. a. Heat map showing significance of differences in transcripts vs. fold change from baseline at 6m post-BNT162b2 and 6m post-CoronaVac. Adjacent to the heatmaps, the log2 fold changes (6-month post-vaccination/pre-vaccination and 1-month post-vaccination/pre-vaccination) are depicted in yellow, alongside the proportion of the cohort exhibiting an increase in gene expression, indicated in red. Only genes that were significantly differentially expressed and overlapped DEGs in the pre-vaccination and 1-month post-vaccination comparison after adjusting for a 5% FDR correction are included. The genes are categorized by their functions: Cellular Process, Immunological Function, and Neural Function. The color code reflects the magnitude of expression, with the intensity of each color correlating to the expression level in each subject. The y-axis in the heatmap is highlighted when genes whose expression levels at M6 are significantly correlated with the relative decrease in sVNTs between 1 and 6 months post-vaccination. Red indicates a positive correlation, while blue indicates a negative correlation. *p value < 0.05. b. Scatter plots showing significantly differential metabolites/ lipids (FDR<0.1) between baseline and M1 and M6. Only the significantly differential features between pre-vaccination and 1-month post vaccination were depicted to present the expression levels at 6-month post-vaccination. The overlapping markers between the two vaccine groups were highlighted in red. P value was given by log2(Abundance)= Time + Sex + Age + 1|ID. C16155: UDP−beta−L−threo−pentopyranos−4−ulose, C18218: 16−Hydroxyhexadecanoic acid, CHEBI177622: Oleoyl-L-α-lysophosphatidic acid. 

Figure 5. Co-Abundance Networks of Omics Features between Longer and Shorter Durability. Network plots illustrate the co-abundance networks that emerged from integrating all differentially expressed omics markers between baseline and 1-month post-vaccination. The features abundance at M1 were used for the correlation calculation. Nodes in these networks are color-coded by omics categories: lipidomics (green), metabolomics (yellow), transcriptomics (orange), and cytokines (purple). Significance is assigned to fair absolute correlations greater than 0.3 (FDR < 0.05).

Figure 6. Multi-omics-constrained mathematical modeling for immune responses. The prediction model was carried out as described in Methods. a. Workflow of the prediction model. b. Model performance in classifying subjects with higher or lower immunogenicity and differentiating between those with longer or shorter durability across the two vaccine groups. 
![image](https://github.com/user-attachments/assets/0fe9c3b9-1ff1-407a-ad01-170ad85ebb7a)

